x-pack/plugins/cloud_security_posture/public/common/constants.ts

```diff
@@ -129,4 +130,17 @@ export const cloudPostureIntegrations: CloudPostureIntegrations = {
       },
     ],
   },
+  vuln_mgmt: {
+    policyTemplate: 'vuln_mgmt',
+    name: 'Vulnerability Management', // TODO: we should use i18n and fix this
+    shortName: 'VULN_MGMT', // TODO: we should use i18n and fix this
+    options: [
+      {
+        type: CLOUDBEAT_VULN_MGMT_AWS,
+        name: 'Amazon Web Services', // TODO: we should use i18n and fix this
```

Comment:
can you elaborate on what exactly not working with i18 here?

Reply:
it is working perfectly fine. it is a temporary and a quick solution to allow cloudbeat's team move forward and testing e2e. So I'm leaving it with hard-coded values to be fixed later on :) 
Which will be fixed by: https://github.com/elastic/kibana/issues/150485